src/main/java/core/basesyntax/strategy/FruitStrategyImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStrategyImpl {
+    public FruitService getFruitService(String operation) {
+        Map<String, FruitService> keyStorage = new HashMap<>();
+        keyStorage.put("b", new Balance());
+        keyStorage.put("p", new Purchase());
+        keyStorage.put("r", new Return());
+        keyStorage.put("s", new Supply());

```

Comment:
Make this map as a field and initialise it in statick block, because you create it all the time when you need to get stratagy